services/graphql/src/graphql/definitions/app-user.js

```diff
@@ -311,6 +337,18 @@ input UpdateAppUserMutationInput {
   payload: UpdateAppUserPayloadInput!
 }
 
+input UpdateAppUserCustomBooleanAnswersMutationInput {
+  "The user id to update."
+  id: String!
+  "The answers to set/update. An empty array will _unset_ all existing answers. A null value will do nothing."
```

Comment:
I'm not sure the empty array will unset all answers now, will it?

Reply:
@zarathustra323  It will not.  On empty or null it will do nothing.